src/pages/library/datasetBuilder/DatasetBuilder.ts

```diff
@@ -46,9 +50,11 @@ interface ObjectSetListItemProps<T extends DatasetBuilderType> {
   value: T;
   checked: boolean;
   onChange: (value: T) => void;
+  icon?: ReactElement;
```

Comment:
This could be anything, not only an icon.. but I'm not sure what a better name for it would be.

Reply:
Maybe we hold off until we are passing in something other than an icon? That might help us know better what the name should be.

Reply:
Super out of scope for this ticket probably but maybe worth having an explicit icon class/react component for all of the Svgs that extends `ReactElement` which would allow is to be more tightly scoped?